kanopi/saplings-component-types
# TW29378050 Media Collection

## Description
Teamwork Ticket(s): [Media Collection Component](https://kanopi.teamwork.com/app/tasks/29378050)

> Add config for new Media Collection Paragraph

## Acceptance Criteria
* User can add a Media Collection component
* Config does not have UUIDs
* Media opens in a modal that has a carousel of all the media in the paragraph

## Steps to Validate
1. Add a Media Collection component
2. Save and confirm it functions as expected
